jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/query/TestQueryCase.java

```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation. All rights reserved.
```

Comment:
Wrong license and license header as such. Use the template from [the Sample](https://github.com/eclipse-ee4j/eclipselink/blob/master/project-admin/Sample.java), please.